src/app/lists/ListDataview.tsx

```diff
@@ -203,15 +206,32 @@ const ListsDataView: React.FC<ListDataViewProps> = ({
                 .delete()
                 .eq("primary_key", itemToDelete.primaryKey);
 
+            const auditLog = {
+                action: "delete a list item",
+                content: {
+                    itemName: itemToDelete.itemName,
+                    itemPrimaryKey: itemToDelete.primaryKey,
+                },
+            } as const satisfies Partial<AuditLog>;
+
             if (error) {
                 const logId = await logErrorReturnLogId(
                     `Error with delete: Ingredient id ${itemToDelete.primaryKey}`,
-                    error
+                    { error: error }
                 );
-                setErrorMsg(error.message + `Error ID: ${logId}`);
-            } else {
-                window.location.reload();
+                await sendAuditLog({
+                    ...auditLog,
+                    wasSuccess: false,
+                    logId,
+                    listId: itemToDelete.primaryKey,
+                });
+                handleSetError(error.message + `Error ID: ${logId}`);
+                return;
             }
+
+            await sendAuditLog({ ...auditLog, wasSuccess: true });
+            setToDelete(null);
+            setToDeleteModalOpen(false);
```

Comment:
I'm not sure if this makes any difference, but conceptually it makes more sense to close the modal without changing its content, and then setting the parcel to be deleted to `null`
```suggestion
            setToDeleteModalOpen(false);
            setToDelete(null);
```